docs/modules/ROOT/pages/user-guide/first-steps.adoc

```diff
@@ -101,6 +115,11 @@ The application has three compile dependencies:
 
 They are managed within the `camel-quarkus-bom` that is imported in `<dependencyManagement>`.
 
+[INFO]
+====
+More about xref:user-guide/dependency-management.adoc[BoMs].
```

Comment:
Is the mixed case `BoMs` intended?

Reply:
Yes, but I do not mind switching to BOM, if you like.

Reply:
I'll leave it for you to decide. It's an acronym, so BOM is more correct. If you don't want to waste time tweaking it, then it's no big deal. PR looks good otherwise :+1: